### PR TITLE
ES: Prueba17072017

### PR DESCRIPTION
**Pull request by NozomiRem**

### CODE REVIEW

File: Story/Episode 1/un_011280.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name01#0,"""???"""
+name02#0,"""???"""
+un_011280_0010#0,"""... ah, <%me>..."""
+un_011280_0020#0,"""Lo siento, estoy bien. Me duele un poco la cabeza. No te preocupes, no estoy tan mal."""
+un_011280_0030#0,"""Gracias por venir, pero creo que necesito descansar. Lo siento""
```
Comment (alama) [inline Story/Episode 1/un_011280.csv]: You forget a `"` here

**Comment by alama** [comment]
Look like copy the EN files, and worked off that

BUT you left some  EN strings as they are, let me see...

File: Story/Episode 1/un_010370.csv [hunk 2]
```diff
@@ -0,0 +1,9 @@
+name01#0,"""？？？"""
+name02#0,"""？？？"""
+un_010370_0010#0,"""Oh, hey compañero! ¿Estás en una mision también? Estoy en medio de una!"""
+un_010370_0020#0,"""Finalmente hemos conseguido acceso a más lugares, así que supongo que este es el comienzo para mí."""
+un_010370_0030#0,"""¿Quieres ayudar? No hay necesidad, ni siquiera sé lo que estoy buscando realmente."""
+un_010370_0040#0,"""No es como si me diera por vencido. Realmente quiero encontrarlo, es algo que creo que necesito hacer yo mismo."""
+un_010370_0050#0,"""¿Cómo lo pongo? Supongo que se siente gratificante o algo para ese efecto."""
+un_010370_0060#0,"""¿O eso realmente importa? De todos modos, si realmente necesito ayuda, la pediré."""
+un_010370_0070#0,"""See you later then!"""
```
Comment (alama) [inline Story/Episode 1/un_010370.csv]: un_010370_0070#0 still in EN?

File: Story/Episode 1/un_010530.csv [hunk 3]
```diff
@@ -0,0 +1,8 @@
+name01#0,"""？？？"""
+name02#0,"""？？？"""
+un_010530_0010#0,"""Hola. Lo siento por suspirar, estoy un poco cansado."""
+un_010530_0020#0,"""It seems like I'm fighting native creatures and not Darkers recently."""
```
Comment (alama) [inline Story/Episode 1/un_010530.csv]: un_010530_0020#0 still in EN?

File: Story/Episode 1/un_011270.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+name01#0,"""？？？"""
+name02#0,"""？？？"""
+un_011270_0010#0,"""¿Cómo soy cuando hablo con otras personas?"""
+un_011270_0020#0,"""Hm, dejame intentarlo."""
+un_011270_0030#0,"""Um, okay."""
```
Comment (alama) [inline Story/Episode 1/un_011270.csv]: un_011270_0030#0 still in EN

File: Story/Episode 1/un_012050.csv [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+un_012050_0060#0,"""Tengo una pregunta para ti."""
+un_012050_0070#0,"""¿Has encontrado la localización del violento Dragonkin Hadred?"""
+un_012050_0080#0,"""¿Realmente no sientes nada?"""
+un_012050_0090#0,"""Sus sentidos para encontrar a otras personas son extremadamente buenos y sin embargo usted no puede sentir su presencia."""
+un_012050_0100#0,"""Lo siento, parece que me adelanté."""
+un_012050_0110#0,"""¿Por qué estás interesado en ese dragón?"""
+un_012050_0120#0,"""Te refieres a Hadred?"""
+un_012050_0130#0,"""Porque ese dragón es un traidor."""
+un_012050_0140#0,"""Hadred Es especialmente fuerte, violento y sobre todo un traidor."""
+un_012050_0150#0,"""Traitors cannot not be forgiven. This is the one iron law of our organization."""
```
Comment (alama) [inline Story/Episode 1/un_012050.csv]: un_012050_0150#0 still in EN

File: Story/Episode 1/un_012180.csv [hunk 6]
```diff
@@ -0,0 +1,8 @@
+un_012180_0010#0,"""¿Que pasa? ¿ya quieres pedirme un autografo?"""
+un_012180_0020#0,"""¿No? Bueno, entonces te importaría no molestarme?"""
+un_012180_0030#0,"""Wait, it's not like I hate you, it's just that it might blow my cover."""
```
Comment (alama) [inline Story/Episode 1/un_012180.csv]: un_012180_0030#0 still in EN

File: Story/Episode 1/un_012310.csv [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+un_012310_0020#0,"""Hola, Tea aqui."""
+un_012310_0030#0,"""La Oficina de Información que no teme el alcance del Escuadrón de la Muerte. ¡Tenemos información fresca para usted!"""
+un_012310_0040#0,"""Resulta que el Mecanismo de Investigación tenía dos funciones; Investigación sobre el otorgamiento de atributos de fotones a organismos vivos, y sobre Darkers."""
+un_012310_0050#0,"""La investigación de fotones a un lado, la investigación Darker es muy sospechoso!"""
+un_012310_0060#0,"""El accidente ocurrió en medio de un experimento, lo que plantea la pregunta, ¿en qué estaban experimentando?"""
+un_012310_0070#0,"""Por desgracia, ahí es donde el sendero se seca! Y realmente quiero saber exactamente lo que pasó!"""
+un_012310_0080#0,"""Pensamos que esto tomaría más tiempo para descubrir, pero cuando preguntamos sobre él nos encontramos con esta información enseguida."""
+un_012310_0090#0,"""Parece como si esta información se filtró a propósito."""
+un_012310_0100#0,"""Hm, Cuando se piensa en ello un poco no es obvio?"""
+un_012310_0110#0,"""From the perspective of those who want to conceal it, a full cover up would look suspicious."""
```
Comment (alama) [inline Story/Episode 1/un_012310.csv]: un_012310_0110#0 still in EN

**Review by TheBlazter — COMMENTED** [review]
It will take me a long time to read (and correct) the 187 files of that PR. On a fast view i detect many errors: missing spaces on words, weird translated lines, and the obvious, 6 lines already on English.

**Pull request merged**